From 5a88ee22d1e68b9292cbbb102c0be81e36b0cee0 Mon Sep 17 00:00:00 2001
From: Tobias <ruck.tobias@gmail.com>
Date: Wed, 17 Jul 2024 18:49:01 +0200
Subject: [PATCH] Update ChainParamsConstants (#29)

Use the `make_chainparams.py` script to update the params.

(cherry picked from commit 311944178a909bcb9aa42d5795094b846ced2be0)

diff --git a/src/networks/abc/chainparamsconstants.cpp b/src/networks/abc/chainparamsconstants.cpp
index 0acb69a9b3ba50773fef6ef428d2aa206e0bb186..4c4382cfade1534f68d4227baa10587c7bcef9e9 100644
--- a/src/networks/abc/chainparamsconstants.cpp
+++ b/src/networks/abc/chainparamsconstants.cpp
@@ -5,13 +5,13 @@
 #include <chainparamsconstants.h>
 
 namespace ChainParamsConstants {
-    const BlockHash MAINNET_DEFAULT_ASSUME_VALID = *BlockHash::FromHex("00000000000000000873a5d6e5c8daa2ea84062059c4d51b9a58cab3d0f65591");
-    const uint256 MAINNET_MINIMUM_CHAIN_WORK = uint256S("000000000000000000000000000000000000000001743eb8a67db277780e550b");
-    const uint64_t MAINNET_ASSUMED_BLOCKCHAIN_SIZE = 214;
-    const uint64_t MAINNET_ASSUMED_CHAINSTATE_SIZE = 3;
+    const BlockHash MAINNET_DEFAULT_ASSUME_VALID = BlockHash::fromHex("f6ff288ef51c0f7b0e27dce0786c282611b84d5b1b967b8ad0edb941599cf8ca");
+    const uint256 MAINNET_MINIMUM_CHAIN_WORK = uint256S("00000000000000000000000000000000000000000000120179e8a2aedc842b57");
+    const uint64_t MAINNET_ASSUMED_BLOCKCHAIN_SIZE = 201;
+    const uint64_t MAINNET_ASSUMED_CHAINSTATE_SIZE = 16;
 
-    const BlockHash TESTNET_DEFAULT_ASSUME_VALID = *BlockHash::FromHex("00000000000a9edccf1337f622f37d5d89a011af1fe9b5367c9582b7ca069712");
-    const uint256 TESTNET_MINIMUM_CHAIN_WORK = uint256S("00000000000000000000000000000000000000000000006ec30ecb9d2bbce6ee");
+    const BlockHash TESTNET_DEFAULT_ASSUME_VALID = BlockHash::fromHex("000000000008afd8c37f77f1bf7316125d98a0ca261827e8db62385228ce36cc");
+    const uint256 TESTNET_MINIMUM_CHAIN_WORK = uint256S("00000000000000000000000000000000000000000000006ead3eb16d8dfb35fe");
     const uint64_t TESTNET_ASSUMED_BLOCKCHAIN_SIZE = 55;
     const uint64_t TESTNET_ASSUMED_CHAINSTATE_SIZE = 2;
 } // namespace ChainParamsConstants
